src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -203,7 +242,9 @@ export const Runtimes = (signal: AbortSignal) => {
         `api/v2/runtimes/${workspaceId}?${qs.stringify({ saturnAutoCreated: true, ...labels })}`,
         _.mergeAll([authOpts(), appIdentifier, { signal }])
       );
-      return res.json();
+      const runtimes = await res.json();
+      const normalizedRuntimes = _.map(getNormalizedListRuntime, runtimes);
```

Comment:
Again not part of this PR, but I wonder why we do the tool label normalizing in listV2 but not listV2WithWorkspace?